nixos/modules/services/web-apps/filebrowser-quantum.nix

```
@@ -0,0 +1,363 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.filebrowser-quantum;
+  jsonFormat = pkgs.formats.json { };
+in
+{
+  options.services.filebrowser-quantum = with lib; {
```

```suggestion
  options.services.filebrowser-quantum = {
```
Try not to `with` large namespaces over large code scopes, it might confuse type check and overall readability. Also it's just 3 letters and a dot 😅